PerfNext/config/benchmarks/data_simple/ODM.xml

```diff
@@ -20,8 +20,6 @@
 				<property name="MAXHEAPSIZE">-Xmx1024m</property>
 				<property name="TENUREDSIZE"></property>
 				<property name="NURSERYSIZE"></property>
-				<property name="MULTITHREAD">4</property>
```

Comment:
Please add description to your commit that you're soft-coding this variable as well.